### ISSUE TITLE
Create build instructions

### ISSUE THREAD

**Issue by doctorlard** [georgd/EB-Garamond#142]
It's pretty simple, just run `make` - but there is a Python module dependency from FontForge that is not quite trivial to set up. I'll put together something for a PR in the next day or two.

**Comment by MrSpinalzo** [comment]
Well, I've totally lost the thread here. Would someone be so kind as to explain me the basic process as if I were six years old?

**Comment by timgrei** [comment]
@MrSpinalzo Sorry, you were in the right thread, I was in the wrong one. 😄

Did you install FontForge?

**Comment by timgrei** [comment]
After installing FontForge find a file in the installation folder called `fontforge-console.bat` and run it. Then open the file `Makefile` in the repo with an editor and change line 15 to `PYTHON?=ffpython`.

**Comment by MrSpinalzo** [comment]
Done. What's next?

**Comment by timgrei** [comment]
Do you know how to put something in the environment path?

**Comment by MrSpinalzo** [comment]
Nope. Don't know what's that.

**Comment by timgrei** [comment]
Okay, let's make sure you even need to do that. Open cmd and type ffpython. If you get a python console, you don't. If you get an error, you do.

**Comment by MrSpinalzo** [comment]
Seems that I do.

**Comment by timgrei** [comment]
Okay. Here is a random link, that tells you where to find the path variable: https://www.howtogeek.com/787217/how-to-edit-environment-variables-on-windows-10-or-11/ You need to add the path to ffpython.exe generated by the bat file to it.

**Comment by MrSpinalzo** [comment]
I'm afraid I'm lost again. There's no ffpython.exe (or maybe I can't find it).

**Comment by timgrei** [comment]
When you open cmd in the folder where the bat was, can you then run ffpython?

**Comment by MrSpinalzo** [comment]
I mean, I typed PYTHON?=ffpython in the makefile, but it didn't generate anything. Or so I think.

**Comment by timgrei** [comment]
No, not the Makefile. The .bat one.

**Comment by MrSpinalzo** [comment]
Oh. I see. I'll try again.
Thanks so much for your patience, by the way.

**Comment by timgrei** [comment]
Your very welcome!

**Comment by MrSpinalzo** [comment]
Let me recapitulate: after running the fontforge.console batch file, I typed PYTHON?=ffpython in the makefile contained in the EB Garamond folder. Then nothing happened, or maybe something happenede but I didn't notice.

**Comment by MrSpinalzo** [comment]
I found, however, a ffpython exe in the bin folder contained in the FontForgeBuilds folder. Is that the exe you were refering to?

**Comment by timgrei** [comment]
Yes! very good. I'm sorry, I'm not on a Windows PC and I have never done this myself, so I'm just reading here and there on Stackoverflow and tell you what I would do.

**Comment by MrSpinalzo** [comment]
Great! I've just added the path to ffpython.exe. What's next now?

**Comment by timgrei** [comment]
Now, you need to install make. There are multiple options for make on Windows. I have never used any of them. Here is one: https://cygwin.com/install.html

**Comment by timgrei** [comment]
In the installer, you apparently need to make a check at make, which is not necessarily default marked.

**Comment by MrSpinalzo** [comment]
Alright. So I installed make. I think.

**Comment by timgrei** [comment]
Now, you don't get your hopes up and go to the repo in cmd and run `make`. If that doesn't work try `make -f Makefile`.

**Comment by MrSpinalzo** [comment]
Shit! It doesn't work.

**Comment by timgrei** [comment]
What does it say?

**Comment by MrSpinalzo** [comment]
The term "make" is not recognized.

**Comment by timgrei** [comment]
You can try to find a make.exe in the Cygwin folder and add that to the path.

**Comment by timgrei** [comment]
I think you see now, why people do this stuff in Linux 😄

**Comment by MrSpinalzo** [comment]
Yeah, I'm getting the idea

**Comment by timgrei** [comment]
@doctorlard @MrSpinalzo I'm able to build on Windows 10 using [Make for Windows](https://gnuwin32.sourceforge.net/packages/make.htm) and the process I described above for ffpython.

**Comment by MrSpinalzo** [comment]
Great! Thank you very much

**Comment by georgd** [comment]
@MrSpinalzo could you get it to work?
@timgrei thanks a lot! I wouldn‘t have been able to give these instructions, lacking an adequate Win10 system. Do you think it’s necessary to document these instructions somewhere?

**Comment by timgrei** [comment]
> @MrSpinalzo could you get it to work? @timgrei thanks a lot! I wouldn‘t have been able to give these instructions, lacking an adequate Win10 system. Do you think it’s necessary to document these instructions somewhere?

I can write instructions, if @doctorlard isn't working on them anyway. I actually think, linux users, whose distro repos don't provide the fontforge python package, have a bigger problem than windows users.

**Comment by georgd** [comment]
Oh, ok. Those aren’t available via pip, are they?

**Comment by timgrei** [comment]
No, that's why the Windows building is so complex, but at least it works. Maybe the Fontforge App builds this ffpython for Linux too, I'll have to look into it.

**Comment by doctorlard** [comment]
Apologies, I've had no time just as I thought I'd be able to contribute something; I was hoping to get something working in Windows first before pushing a PR.
